### PR TITLE
Prevent  recipe details render

### LINKED ISSUE #60: Stop recipe details from rendering when deleting a recipe
## Expected Behavior

Should be able to delete a saved recipe from the sidebar without any other action from the page.

## Actual/Current Behavior

Recipe details is rendering when a saved recipe is deleted from the sidebar.

## Steps to Reproduce the Problem

  1.  Save a recipe to the saved recipes list.
  2. Delete recipe from list and observe if the recipe details opens. 

## Testing Steps

_How do the users test this change?_

  1. Save a recipe to the saved recipes list.
  2. Delete recipe from list and observe if the recipe details opens.

### PR DESCRIPTION
**Pull request by Gjhnsn**
## Changes
1. Prevent recipe details from rendering when a recipe is deleted from the saved list on the sidebar.

## Purpose
So the recipe details page does not open when a user deletes a saved recipe.

## Approach
stopPropagation on the event

Closes #60

### CODE REVIEW

File: src/components/Sidebar.tsx [hunk 1]
```diff
@@ -198,7 +192,11 @@ const Sidebar: FC<SidebarProps> = ({
     }
   };
 
-  const removeRecipe = (recipeID: number) => {
+  const removeRecipe = (
+    e: React.MouseEvent<HTMLDivElement>,
```
Comment (dannysanchez559) [inline src/components/Sidebar.tsx:196]: never seen this before.....

**Review by dannysanchez559 — APPROVED** [review]
good job

File: src/components/Sidebar.tsx (hunk 1 above)
Comment (Lap343) [inline src/components/Sidebar.tsx:196]: interesting

**Review by BlackCubes — APPROVED** [review]
Looks clean!

File: src/components/Sidebar.tsx [hunk 2]
```diff
@@ -198,7 +192,11 @@ const Sidebar: FC<SidebarProps> = ({
...
+    recipeID: number
+  ) => {
+    e.stopPropagation();
```
Comment (kmausolf) [inline src/components/Sidebar.tsx:199]: Good call!  I'm guessing you have clickable containers on top of clickable containers?

**Review by Lap343 — APPROVED** [review]
Nice job!

File: src/components/Sidebar.tsx (hunk 2 above)
Comment (Gjhnsn) [reply src/components/Sidebar.tsx:199]: Yup, sure do!

**Pull request merged**